crates/turborepo-telemetry/src/events.rs

```diff
@@ -0,0 +1,40 @@
+use serde::{Deserialize, Serialize};
+
+/// All possible telemetry events must be included in this enum
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub enum TelemetryEvent {
+    Fallback(Fallback),
+    KeyVal(KeyVal),
+}
+
+/// Individual events are defined here
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct KeyVal {
+    key: String,
```

Comment:
If I'm understanding this correctly and we want to prevent random strings from being used as keys, then we can use an `enum` here.

```
#[derive(Debug, Clone, Serialize, Deserialize)]
enum Key {
    Framework,
    Command,
}
```

```suggestion
    key: Key,
```

Reply:
These were mostly for convenience, but this is _nicer_